src/pages/CreateBillableMetric.tsx

```diff
@@ -244,6 +244,17 @@ const CreateBillableMetric = () => {
                         </ComboboxHeader>
                       ),
                     }}
+                    renderGroupInputStartAdornment={[
+                      {
+                        value: translate('text_6310755befed49627644222b'),
```

Comment:
Hum... Why not just use an object like in `renderGroupHeader` to be consistent ?

```
{
  [groupKey]: "translation",
  ...
}
```

Reply:
Indeed, way better ! I updated the code with this approach